.gitlab-ci.yml

```diff
@@ -27,9 +29,26 @@ default:
       .gitlab/assert_ats_results.awk -v PASSED=22 FAILED=12 SKIPPED=10
       test/HelloATS/$SYS_TYPE*.logs/ats.log
 
+.hello_Flux:
+  extends: .install_ATS
+  stage: test
+  script:
+    - cd test/HelloATS
+    - mpicc hello_ats.c
+    - python3 create_test_ats.py
+    - atsflux --account=cbronze test.ats
```

Comment:
@wihobbs in response to your note on this. We would have more flexibility with
`- atsflux --account=${BANK} test.ats`
after adding the variable to this .hello_Flux: job. Something like...
```
.hello_Flux:
   # <extends: and stage: lines...>
   variables:
     BANK: cbronze
  # <script: lines ...>
```

There are ways to set variables like this outside of these yaml files and this would leave room to do so if needed.

Reply:
Nice! I like this. Let me fix and push again.